app/services/case_filter/case_type_filter.rb

```diff
@@ -0,0 +1,92 @@
+module CaseFilter
+  class CaseTypeFilter < CaseMultiChoicesFilterBase
+
+    SUB_FILTER_MAP = {
+      FOI: ['foi-standard',
+            'foi-ir-compliance',
+            'foi-ir-timeliness', 
+            'overturned-ico'],
+      SAR: ['sar-non-offender', 'overturned-ico'],
+      OFFENDER_SAR: ['offender-sar'],
+      OFFENDER_SAR_COMPLAIN: ['offender-sar-complain'],
+      ICO: ['ico-appeal']
+    }.with_indifferent_access.freeze
+
+    class << self
+      def filter_attributes
+        [:filter_case_type]
+      end    
+
+    end
+
+    def get_available_choices
```

Comment:
Here again it would be nice to match the interface of either `available_choices` or `get_available_choices`